pkg/cmd/server/start/master_args.go

```diff
@@ -486,7 +486,17 @@ func (args MasterArgs) GetServerCertHostnames() (util.StringSet, error) {
 		return nil, err
 	}
 
-	allHostnames := util.NewStringSet("localhost", "127.0.0.1", "openshift.default.svc.cluster.local", "kubernetes.default.svc.cluster.local", masterAddr.Host, masterPublicAddr.Host, assetPublicAddr.Host)
+	allHostnames := util.NewStringSet(
+		"localhost", "127.0.0.1",
+		"openshift.default.svc.cluster.local",
+		"openshift.default.svc",
+		"openshift.default",
+		"openshift",
+		"kubernetes.default.svc.cluster.local",
```

Comment:
not in the command this is run from (this is the default stuff you get running `openshift start`). if you edit your config to change dns suffixes, you'll need to regen your server cert with the custom suffix